dockers/docker-sflow/supervisord.conf

```diff
@@ -28,6 +28,17 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
 
+[program:start]
+command=/usr/bin/start.sh
+priority=2
+autostart=false
```

Comment:
>false [](http://example.com/codeflow?start=10&length=5)

true?

Reply:
Correct me if I wrong but I guess it is not required as the dependent_startup flag is true. So it will eventually gets started post dependent-startup. Also, I have referred other supervisord.conf files and found that it is most of the times false only.